src/util/mask.js

```diff
@@ -13,22 +18,50 @@ const allowedMasks = {
   }
 }
 
+/**
+ * Default delimiter regexp
+ * 
+ * @type {RegExp}
+ */
+const defaultDelimiter = /[-!$%^&*()_+|~=`{}\[\]:";'<>?,.\/ ]/
```

Comment:
maybe this const could be used in mask.js/allowedMasks and maskable.js/data/delimiters?